custom_components/kia_uvo/requirements.txt

```diff
@@ -1 +1 @@
-hyundai_kia_connect_api==3.44.5
+hyundai_kia_connect_api==3.48.0
```

Comment:
```suggestion
hyundai_kia_connect_api==3.48.1
```

Reply:
while we have a requirements.txt to be consumed, we did not set this library properly, so, it is coming from this file for home assistant: https://github.com/Hyundai-Kia-Connect/kia_uvo/blob/master/custom_components/kia_uvo/manifest.json